2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,49 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+
+let chicken = {
+  title:"chicken",
+  serves:4,
+  ingredients: ["chicken","potatoes"]
+}
+
+let pork = {
+  title:"pork",
+  serves:4,
+  ingredients: ["pork","tomtoes"]
+}
+
+let steak = {
+  title:"steak",
+  serves:2,
+  ingredients: ["beef","pepper"]
+}
+
+let lamb = {
+  title:"lamb",
+  serves:3,
+  ingredients: ["lamb","garlic"]
+}
+
+let fish = {
+  title:"fish",
+  serves:6,
+  ingredients: ["fish","onion"]
+}
+console.log(Object.entries(fish));
+
+function printRecipe(obj) {
```

Comment:
this is really good and it's clear you have a good understanding of accessing object keys & values. The question asks for each recipe to be in this format when console.log is used:
```
Mole
  Serves: 2
  Ingredients:
  cinnamon
  cumin
  cocoa
```
Can you think of ways you might change your approach so that it follows this format? (for example not having the title key logged & having the ingredients on a new line)